packages/grid/x-data-grid/src/hooks/features/virtualization/useGridVirtualScroller.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
```

Comment:
It looks like there's no column virtualization anymore, you can see the performance taking a hit in https://deploy-preview-10059--material-ui-x.netlify.app/x/react-data-grid/virtualization/#column-virtualization

Reply:
Forgot to reenable it after making changes :| It's back in.

Reply:
Now we're talking! 🚀